packages/solutions/app-tools/src/analyze/nestedRoutes.ts

```diff
@@ -194,24 +194,24 @@ export const walk = async (
 
     if (itemWithoutExt === NESTED_ROUTE.LAYOUT_LOADER_FILE) {
       if (!route.loader) {
-        route.loader = itemPath;
+        route.loader = replaceWithAlias(alias.basename, itemPath, alias.name);
```

Comment:
why not up the code `replaceWithAlias(alias.basename, itemPath, alias.name)` to upper scope.
It seems use the same params to get same result

Reply:
done~